sound.js

```diff
@@ -31,7 +39,7 @@ function Sound(filename, basePath, onError) {
   this._pan = 0;
   this._numberOfLoops = 0;
   this._speed = 1;
-  RNSound.prepare(this._filename, this._key, (error, props) => {
+  RNSound.prepare(this._filename, this._key, enableSMTCIntegration, (error, props) => {
```

Comment:
Whoops, can't believe I forgot to update iOS and Android!

But yes that's a great idea, I will replace this an options object so that it can be extended in the future. And will also update iOS and Android!